CONTRIBUTING.md

```diff
@@ -7,17 +7,12 @@
         $ git checkout -b <branch>
 
 3.  Add new functions as separate source files in src/. If the function is
```

Comment:
should we add a line/subsection something like "Test files should be included for all functions. Include your test files in the `test` directory."